ridemyway/tests/integration/vehicle/test_vehicle.py

```diff
@@ -0,0 +1,47 @@
+from ridemyway.tests.base_test_case import VehicleBaseTestCase
+
+
+class TestVehicle(VehicleBaseTestCase):
+    def test_create_vehicle(self):
+        login_uri = "/api/v1/auth/users/login/"
+        params_admin_user = {
+            "email":  self.admin_user.email,
+            "password": "password123"
+        }
+
+        self.set_authorization_header(login_uri, params_admin_user)
+
+        vehicle_params = {
```

Comment:
For this data, we have a file under `tests/test_data.py` where we can declare it from and import it, for multiple usages.

Reply:
Done